### PR TITLE
Identity UI fix razor content

### PR DESCRIPTION
**Pull request by HaoK**
The identity UI files are currently incorrectly marked as content after the bootstrap sxs changes

Part of https://github.com/dotnet/aspnetcore/issues/36288

### CODE REVIEW

**Comment by HaoK** [comment]
Looks like this worked, as the nullable issues no longer trigger

File: src/Identity/UI/src/Areas/Identity/Pages/V5/Account/Login.cshtml [hunk 1]
```diff
@@ -67,7 +67,7 @@
                     <form id="external-account" asp-page="./ExternalLogin" asp-route-returnUrl="@Model.ReturnUrl" method="post" class="form-horizontal">
                         <div>
                             <p>
-                                @foreach (var provider in Model.ExternalLogins!)
+                                @foreach (var provider in Model.ExternalLogins)
```
Comment (HaoK) [inline src/Identity/UI/src/Areas/Identity/Pages/V5/Account/Login.cshtml:70]: This used to cause publish nullability warnings in the project template tests

**Review by javiercn — APPROVED** [review]

File: src/Identity/UI/src/Microsoft.AspNetCore.Identity.UI.csproj [hunk 2]
```diff
@@ -23,6 +23,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Content Update="**/*.cshtml" Pack="false" />
```
Comment (pranavkm) [inline src/Identity/UI/src/Microsoft.AspNetCore.Identity.UI.csproj:26]:
```suggestion
    <Content Update="**\*.cshtml" Pack="false" />
```

Not that it matters, but for consistency with all the other globs.

**Review by pranavkm — APPROVED** [review]

**Comment by HaoK** [comment]
@dotnet/aspnet-build could I get this merged, looks like everything has passed except for the (now skipped) mac OSx queue

**Pull request merged**